Composer/packages/client/src/components/AddRemoteSkillModal/helper.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import formatMessage from 'format-message';
+import { luIndexer, combineMessage } from '@bfc/indexers';
+
+import httpClient from '../../utils/httpUtil';
+import TelemetryClient from '../../telemetry/TelemetryClient';
+
+export const importOrchestractor = async (projectId: string, reloadProject, setApplicationLevelError) => {
+  const reqBody = {
+    package: 'Microsoft.Bot.Builder.AI.Orchestrator',
+    version: '',
+    source: 'https://botbuilder.myget.org/F/botbuilder-v4-dotnet-daily/api/v3/index.json',
+    isUpdating: false,
+    isPreview: true,
+  };
+  try {
+    const results = await httpClient.post(`projects/${projectId}/import`, reqBody);
+    // check to see if there was a conflict that requires confirmation
+    if (results.data.success === false) {
+      TelemetryClient.track('PackageInstallConflictFound', { ...reqBody, isUpdate: reqBody.isUpdating });

```

Comment:
are we gonna let the user know it failed? because of a conflict?

Reply:
we show error through setApplicationLevelError

Reply:
I meant when we are at this line, all we do is to track it via telemetry, we are not showing anything to the user to basically say: it was unsuccessful because there was a conflict

Reply:
Thanks, now we show confirm dialog.